app/services/collection_observer.rb

```diff
@@ -66,9 +66,10 @@ def self.collection_managers_excluding_owner(work_version)
   private_class_method :collection_managers_excluding_owner
 
   def self.mailer_with_owner(user:, work_version:)
-    owner = work_version.work.owner
-    collection = work_version.work.collection
-    CollectionsMailer.with(user:, collection_version: collection.head, owner:)
+    work = work_version.work
+    owner = work.owner
+    collection_version = work.collection.head
+    CollectionsMailer.with(user:, collection_version:, owner:, work:)
   end
```

Comment:
some small refactoring here, and make sure to pass the `work` along to the mailer, which now needs it